packages/react/.changeset/little-bikes-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@gadgetinc/react": minor
+---
+
+Added support for Upsert actions in AutoForm
```

Comment:
Can you add an example of how to use it so we know what to add to the docs?

Reply:
Updated with examples showing usage